ext/bcmath/libbcmath/src/raisemod.c

```diff
@@ -57,6 +57,14 @@ raise_mod_status bc_raisemod(bc_num base, bc_num expo, bc_num mod, bc_num *resul
 		return MOD_IS_ZERO;
 	}
 
+	/* Any integer number mod 1 must be equal to 0 */
+	// TODO What about mod -1?
```

Comment:
Of course, it should be 0. Therefore, I think it is better to use `_bc_do_compare` to compare ignoring the sign, instead of `bc_compare`.

But I guess that can be changed in a follow-up PR.

Reply:
That would also need to be backported as that's an existing bug. I'll first merge the fix, then this PR after a rebase.